deps/getdns/getdns.gyp

```diff
@@ -0,0 +1,149 @@
+            '../openssl/openssl.gyp:openssl'
+          ]
+        }],
+        ['node_shared_libuv=="false"', {
+          'dependencies': [
+            '../uv/uv.gyp:libuv'
+          ]
+        }],
+        [ 'OS=="linux"', {
+          'defines': [
+            '_GNU_SOURCE',
+            '_DEFAULT_SOURCE',
+          ],
+          'sources': [
+            'getdns/src/compat/arc4random.c',
+            'getdns/src/compat/explicit_bzero.c',
+            'getdns/src/compat/arc4_lock.c',
+            'getdns/src/compat/arc4random_uniform.c',
+            'getdns/src/compat/strlcpy.c',
+          ],
+        }],
+        [ 'OS=="win"', {
+          'sources': [
+            'getdns/src/compat/gettimeofday.c',
+            'getdns/src/compat/arc4random.c',
+            'getdns/src/compat/arc4_lock.c',
+            'getdns/src/compat/arc4random_uniform.c',
+            'getdns/src/compat/getentropy_win.c',
+            'getdns/src/compat/strlcpy.c',
+            'getdns/src/compat/mkstemp.c',
+            'getdns/src/compat/strptime.c',
+          ],
+        }],
+        [ 'OS=="freebsd"', {
+          'defines': [
+            '_POSIX_C_SOURCE=200112L',
+            '_XOPEN_SOURCE=600',
+          ],
+        }],
+        [ 'OS=="solaris"', {
+          'defines': [
+            '__EXTENSIONS_',
+          ],
+        }],
+        [ 'OS in "mac ios"', {
+          'defines': ['_DARWIN_C_SOURCE'],
+        }],
+      ],
+      'sources': [
+        'getdns/src/anchor.c',
+        'getdns/src/const-info.c',
+        'getdns/src/convert.c',
+        'getdns/src/context.c',
+        'getdns/src/dict.c',
+        'getdns/src/dnssec.c',
+        'getdns/src/general.c',
+        'getdns/src/list.c',
+        'getdns/src/request-internal.c',
+        'getdns/src/mdns.c',
+        'getdns/src/platform.c',
+        'getdns/src/pubkey-pinning.c',
+        'getdns/src/rr-dict.c',
+        'getdns/src/rr-iter.c',
+        'getdns/src/server.c',
+        'getdns/src/stub.c',
+        'getdns/src/sync.c',
+        'getdns/src/ub_loop.c',
+        'getdns/src/util-internal.c',
+
+        'getdns/src/extension/select_eventloop.c',
+        'getdns/src/extension/libuv.c',
+
+        'getdns/src/gldns/keyraw.c',
+        'getdns/src/gldns/gbuffer.c',
+        'getdns/src/gldns/wire2str.c',
+        'getdns/src/gldns/parse.c',
+        'getdns/src/gldns/parseutil.c',
+        'getdns/src/gldns/rrdef.c',
+        'getdns/src/gldns/str2wire.c',
+
+        'getdns/src/util/rbtree.c',
+        'getdns/src/util/lruhash.c',
+        'getdns/src/util/lookup3.c',
+        'getdns/src/util/locks.c',
+
+        'getdns/src/jsmn/jsmn.c',
+
+        'getdns/src/yxml/yxml.c',
+
+        'getdns/src/tls/val_secalgo.c',
+        'getdns/src/tls/anchor-internal.c',
+
+        'getdns/src/openssl/tls.c',
+        'getdns/src/openssl/pubkey-pinning-internal.c',
+        'getdns/src/openssl/keyraw-internal.c',
+
+        '<(SHARED_INTERMEDIATE_DIR)/getdns/src/version.c',
```

Comment:
```suggestion
```

I think `process_outputs_as_sources` makes this redundant. Removing the line fixes the ninja build.